_delphi_utils_python/.pylintrc

```diff
@@ -1,8 +1,22 @@
-[DESIGN]
```

Comment:
whats going on with the util files changing?

Reply:
In order to use GeoMapper, I had to "merge" the main `delphi_utils` folder into the current `sensorization` branch. This PR merges the `sensorization` branch into the `dv-package` branch, which is using a very old version of `delphi_utils`

Reply:
> I had to "merge" the main delphi_utils folder

uh does this mean if we ever merge the dv-package branch into main, it'll generate an enormous pile of conflicts? ...wouldn't it be better to just `git merge main`?

Reply:
I didn't think it would since I used `git checkout main _delphi_utils_python` but my knowledge of git is very limited. I can do a `git merge main` instead

Reply:
Thanks for clarifying! I wasn't familiar with that syntax, so just to be sure, I have double-checked by doing this:
```
git checkout sensorization
git checkout -b test_merge
git merge main
```
and there are indeed no conflicts. you're all set!